src/app/profiles/profile-detail/profile-detail.component.ts

```diff
@@ -163,14 +161,20 @@ export class ProfileDetailComponent implements OnInit {
   networkConfigChange (value: boolean): void {
     if (!value) {
       this.profileForm.controls.ciraConfigName.disable()
+      this.wifiAutocomplete.reset({ value: '', disabled: true })
       this.profileForm.controls.ciraConfigName.setValue(null)
+      this.selectedWifiConfigs = []
```

Comment:
This is a bit jarring, especially if i've taken the time to add and prioritize all my wifi profiles.  Let's remove this and instead add a validator that will prevent save if static is chosen.

Reply:
Updated